source/identity-access-management/roles.rst

```diff
@@ -117,6 +117,213 @@ manage their own account information. This role cannot view, create or destroy
+   |                       | openstack.volume.delete_transfer                                       |
+   |                       | openstack.volume.get_all_transfers                                     |
+   |                       | openstack.backup.create                                                |
+   |                       | openstack.backup.delete                                                |
+   |                       | openstack.backup.get                                                   |
+   |                       | openstack.backup.get_all                                               |
+   |                       | openstack.backup.restore                                               |
+   |                       | openstack.snapshot_extension.snapshot_actions.update_snapshot_status   |
+   |                       |                                                                        |
+   |                       | ORCHESTRATION SERVICE                                                  |
+   |                       | openstack.stacks.lookup                                                |
+   +-----------------------+------------------------------------------------------------------------+
+   | Authentication Only   | openstack.keypair.create                                               |
+   |                       | openstack.quota.show                                                   |
+   +-----------------------+------------------------------------------------------------------------+
+   | Project Administrator | openstack.volume.get                                                   |
+   |                       | openstack.volume.initialize_connection                                 |
+   |                       | keystone.identity.project_users_access                                 |
+   +-----------------------+------------------------------------------------------------------------+
+   | Project Moderator     | keystone.identity.project_users_access                                 |
+   +-----------------------+------------------------------------------------------------------------+
+   | Compute Start/Stop    | openstack.compute.start                                                |
+   |                       | openstack.compute.stop                                                 |
+   |                       | openstack.compute.shelve                                               |
+   |                       | openstack.compute.unshelve                                             |
+   +-----------------------+------------------------------------------------------------------------+
+   | Heat Stack Owner      | openstack.Orchestration.*                                              |
+   +-----------------------+------------------------------------------------------------------------+
+   | Object Storage        | Swift.*                                                                |
```

Comment:
Ditto

Reply:
I'm not sure where to find the swift policy as it wasn't included with the rest of the polices that i was shown.